src/symbol/observable.ts

```diff
@@ -0,0 +1,18 @@
+import {root} from '../util/root';
+
+const Symbol: any = root.Symbol;
+
+export let $$observable: any;
+
+if (typeof Symbol === 'function') {
+  if (!Symbol.observable) {
+    if (typeof Symbol.for === 'function') {
+      $$observable = Symbol.for('observable');
+    } else {
+      $$observable = Symbol('observable');
```

Comment:
Well, wait, since this is patching `Symbol.observable` on the following line, any other properly implemented `Symbol.observable` module like this should return that value, as they should always check to see if they exist first before they try to get something else.